pkg/skaffold/initializer/init.go

```diff
@@ -288,9 +288,12 @@ func detectBuilders(enableJibInit, enableBuildpackInit bool, path string) ([]Ini
 	}
 
 	// Check for Dockerfile
-	if docker.Validate(path) {
-		results := []InitBuilder{docker.ArtifactConfig{File: path}}
-		return results, true
+	base := filepath.Base(path)
+	if strings.Contains(base, "Dockerfile") {
```

Comment:
FWIW, I've also seen a lot of `*.dockerfile` out there

Reply:
so, it would be better to make it case insensitive?

Reply:
I've made it case insensitive and updated test to cover this case.